gen-ai-orchestrator-server/Dockerfile

```diff
@@ -1,13 +1,32 @@
 FROM python:3.11-alpine
 
+# Set environment variables
+ENV PYTHONUNBUFFERED=1 \
+    POETRY_VERSION=1.8.3 \
+    # Poetry virtual environments are disabled to install packages globally inside the container.
+    POETRY_VIRTUALENVS_CREATE=false \
+    POETRY_NO_INTERACTION=1 \
+    # The Poetry cache directory is set to reduce repeated downloads in future builds.
+    POETRY_CACHE_DIR='/var/cache/pypoetry'
+
+# Install Poetry
+RUN pip install "poetry==$POETRY_VERSION"
```

Comment:
I'm not sure fabrikIO correctly support it we it would be better do use multi-staged docker file : https://medium.com/@albertazzir/blazing-fast-python-docker-builds-with-poetry-a78a66f5aed0#:~:text=Using%20Docker%20multi%2Dstage%20builds

Reply:
Yes, I'm working on it

Reply:
I'm having a docker build problems  (proxy/repository). It would be better to deal with this later.